cmd/oras/root/push.go

```diff
@@ -152,6 +162,7 @@ Example - Push file "hi.txt" into an OCI image layout folder 'layout-dir' with t
 	cmd.Flags().IntVarP(&opts.concurrency, "concurrency", "", 5, "concurrency level")
 	opts.SetTypes(option.FormatTypeText, option.FormatTypeJSON, option.FormatTypeGoTemplate)
 	option.ApplyFlags(&opts, cmd.Flags())
+	cmd.MarkFlagsMutuallyExclusive("config", "platform")
```

Comment:
Use `CheckMutuallyExclusiveFlags` instead. The output of this function is hard to read.

Reply:
Done.